internal/http/services/owncloud/ocdav/proppatch.go

```diff
@@ -109,13 +109,9 @@ func (s *svc) handleSpacesProppatch(w http.ResponseWriter, r *http.Request, spac
 		return status, err
 	}
 
-	// retrieve a specific storage space
-	ref, rpcStatus, err := spacelookup.LookUpStorageSpaceReference(ctx, c, spaceID, r.URL.Path, true)
-	if err != nil {
-		return http.StatusInternalServerError, err
-	}
-	if rpcStatus.Code != rpc.Code_CODE_OK {
-		return rstatus.HTTPStatusFromCode(rpcStatus.Code), errtypes.NewErrtypeFromStatus(rpcStatus)
+	ref := spacelookup.MakeStorageSpaceReference(spaceID, r.URL.Path)
+	if ref == nil {
```

Comment:
The reference will never be nil.

Reply:
We are in control of the `MakeStorageSpaceReference` function. When we change the implementation to also return nil we will know. IMO it's really bad to leave dead code here. It's just bloating the code.

Reply:
returning nil now